src/moneywiz_api/model/transaction.py

```diff
@@ -11,6 +11,10 @@
 from moneywiz_api.types import ID
 
 
+TOLERANCE = 1e-8
+TOLERANCE_AMOUNT = 0.01
```

Comment:
can we keep this as 0.001? for the sake of backward compatibility.

Reply:
I'll check with my DB and get back to you.

Reply:
My base currency is USD (the amount is rounded to 2 decimal digits). This transaction is in ETH (the original amount has 8 decimal digits), and the type is an expense:
`-0.00116681 * 1840.35755115 = -2.147348`

0.001 is not enough, but 0.005 worked.